src/main/resources/db/migration/all/20231009162205__add_support_for_cas3_to_domain_events_table.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE domain_events ALTER COLUMN application_id DROP NOT NULL;
+
+ALTER TABLE domain_events ADD COLUMN booking_id UUID;
+ALTER TABLE domain_events ADD COLUMN service TEXT;
+
+UPDATE domain_events
+SET service = 'CAS1';
```

Comment:
[It doesn't appear like CAS2 have merged their domain event work](https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/981) in yet so this should be safe for all environments. @Harriethw FYI this would be a change to consider ontop of that draft.